File: eggs/KitKat/src/main/java/com/android_k/egg/preview/SnapshotProvider.java
package com.android_k.egg.preview;


import android.content.Context;
import android.view.Gravity;
import android.view.View;
import android.widget.FrameLayout;
import android.widget.ImageView;

import com.android_k.egg.R;


public class SnapshotProvider extends com.dede.basic.provider.SnapshotProvider {

    @Override
    public View create(Context context) {
        FrameLayout mContent = new FrameLayout(context);

        final FrameLayout.LayoutParams lp = new FrameLayout.LayoutParams(
                FrameLayout.LayoutParams.WRAP_CONTENT,
                FrameLayout.LayoutParams.WRAP_CONTENT);
        lp.gravity = Gravity.CENTER;

        final ImageView logo = new ImageView(context);
        logo.setImageResource(R.drawable.k_platlogo_preview);
        logo.setScaleType(ImageView.ScaleType.CENTER_INSIDE);

        mContent.addView(logo, lp);
        return mContent;
    }
}

File: eggs/T/src/com/android_t/egg/SnapshotProvider.kt
package com.android_t.egg

import android.content.Context
import android.view.Gravity
import android.view.View
import android.widget.FrameLayout
import android.widget.ImageView
import com.dede.basic.provider.SnapshotProvider

class SnapshotProvider : SnapshotProvider() {

    override fun create(context: Context): View {
        val layout = FrameLayout(context)

        val logo = ImageView(context)
        logo.setImageResource(R.drawable.t_platlogo)
        val lp: FrameLayout.LayoutParams = FrameLayout.LayoutParams(
            FrameLayout.LayoutParams.MATCH_PARENT,
            FrameLayout.LayoutParams.MATCH_PARENT
        )
        lp.gravity = Gravity.CENTER
        layout.addView(logo, lp)

        return layout
    }
}